### PR TITLE
[WPT sync task] Rename PR branches to "webref/*"

### PR DESCRIPTION
**Pull request by tidoust**
Branches created on Web Platform Tests are still named after `reffy-reports/*`. Following the rename to `webref`, this update changes the name of created branches to `webref/*`.

As existing PRs are identified through the name of the branch, the code still checks for and handles existing `reffy-reports/*` pull requests. See discussion in https://github.com/w3c/webref/pull/53

### CODE REVIEW

File: wpt-sync/sync.js [hunk 1]
```diff
@@ -166,21 +167,23 @@ async function updatePullRequests(dir, localBranches, remote, remoteBranches) {
 
     console.log('Updating pull requests:');
     for (const branch of localBranches) {
-        assert(branch.startsWith('reffy-reports/'));
+        assert(branch.startsWith('webref/'));
+        const oldBranch = branch.replace(/webref/, 'reffy-reports');
```
Comment (foolip) [inline wpt-sync/sync.js:171]:
```suggestion
        const oldBranch = branch.replace(/^webref/, 'reffy-reports');
```

Just to be sure :)

**Review by foolip — APPROVED** [review]
Looks like this will work. Can it be tested before going live?

**Comment by tidoust** [comment]
> Looks like this will work. Can it be tested before going live?

I was sort of hoping that you would be able to do it :)
I tested the individual git commands but haven't tried to run the script. I'll give it a try.

**Comment by tidoust** [comment]
I would say that this seems to work _well enough_ in that the script correctly creates `webref/*` branches when needed and seems to be willing to run the right push commands against the wpt repository. I'm not sure how to test that all cases are handled correctly though. Do you think that we can go ahead and merge, or is there an easy way to run more thorough tests?

**Comment by foolip** [comment]
@tidoust yeah, let's just land this and see what breaks :)

**Comment by tidoust** [comment]
FWIW, it did break ;) There were a couple of places where the code didn't account for the fact the local and remote branches might not have the same name. Fixed in https://github.com/w3c/webref/commit/700fd9ce6548f0f861bfbe8714de53708a4c01ef

The job still fails, but with the same "Cannot lock ref" error as reported in #54 (so a priori not something related to this PR). I see that the job still manages to create PRs on WPT though...

**Pull request merged**